test/e2e/common/setup-kubestellar.sh

```diff
@@ -90,23 +90,13 @@ echo "wds1 created."
 : -------------------------------------------------------------------------
 : Run OCM transport controller in a pod
 :
-cd "${SRC_DIR}/../../.." ## go up to KubeStellar directory
-KUBESTELLAR_DIR="$(pwd)"
-OCM_TRANSPORT_PLUGIN_RELEASE="0.1.0-rc4"
-curl -sL https://github.com/kubestellar/ocm-transport-plugin/archive/refs/tags/v${OCM_TRANSPORT_PLUGIN_RELEASE}.tar.gz | tar xz
-cd ocm-transport-plugin-${OCM_TRANSPORT_PLUGIN_RELEASE}
-OCM_TRANSPORT_PLUGIN_DIR="$(pwd)"
-pwd
-echo "replace github.com/kubestellar/kubestellar => ${KUBESTELLAR_DIR}/" >> go.mod
-go mod tidy # TODO to be deleted next time we bump ocm transport release (done in ocm transport makefile)
-IMAGE_TAG=${OCM_TRANSPORT_PLUGIN_RELEASE} make ko-build-local
-kind load --name kubeflex docker-image ko.local/transport-controller:${OCM_TRANSPORT_PLUGIN_RELEASE} # load local image to kubeflex
-cd "${KUBESTELLAR_DIR}"
-pwd
-rm -rf ${OCM_TRANSPORT_PLUGIN_DIR}
-echo "running ocm transport plugin..."
-kubectl config use-context kind-kubeflex ## transport deployment script assumes it runs within kubeflex context
-IMAGE_PULL_POLICY=Never ./scripts/deploy-transport-controller.sh wds1 imbs1 ko.local/transport-controller:${OCM_TRANSPORT_PLUGIN_RELEASE}
+OCM_TRANSPORT_PLUGIN_RELEASE="0.1.1"
+
+kflex ctx # root
+
+helm upgrade --install ocm-transport-plugin oci://ghcr.io/kubestellar/ocm-transport-plugin/chart/ocm-transport-plugin --version ${OCM_TRANSPORT_PLUGIN} \
+--set transport_cp_name=imbs1 \
+--set wds_cp_name=wds1
```

Comment:
This will not work.
the e2e setup should build local image from the OTP repo.
explanation:
OTP uses ks/ks functions. if we now change any of the ks/ks transport related functions, we need to build locally the OTP image. otherwise we don't test the latest change.

Reply:
Since OTP builds its own image and chart, should those be separately tested in the OTP repo.
Kubestellar is importing an existing OTP chart for use.
I believe this is not different than KS using kflex chart... and not building kflex locally.
Or not build OCM locally, etc etc
Since we are advocating for separation, a KS test should not require a OTP build, in my opinion.
May be we can discus this in one of the meetings.

Reply:
I agree with you on the general idea. 
I agree that KS test should **NOT** require OTP local build. 
but in reality OTP currently has some issues in terms of the dependencies between KS and OTP. things have to be improved and separated better. 
unfortunately, until that happens there is no good way to test KS without OTP local build.

Reply:
@francostellari Currently our e2e is aimed for the developer - it builds the local KS , while using exiting packages of the other components. 
While OTP resides on different repo and is has it's own image - it's not really an "external" package and the design today is that it needs to be build against a specific KS version.  So due to this dependency, for the testing of latest KS you do need to build OTP against that code which means you can't use pre-built image

Reply:
BTW, we are planning to add e2e for releases, for that e2e we will use release images for both KS and OTP

Reply:
In the future we are discussing with @nirrozenbaum how to change this OTP <--> KS dependency so you don't need to build OTP against the KS code.

Reply:
@ezrasilvera so, in conclusion, I should change the user facing instructions to use Helm but the e2e will remain as is testing an alternative process. I will revert the changes with the exception of examples.md. Right?

Reply:
> @ezrasilvera so, in conclusion, I should change the user facing instructions to use Helm but the e2e will remain as is testing an alternative process. I will revert the changes with the exception of examples.md. Right?

@francostellari Yes.  BTW, @effi-ofer is developing e2e that tests releases, those tests can use helm.